local/config.yaml

```diff
@@ -8,4 +8,6 @@ grafeas:
   storage_type: elasticsearch
   elasticsearch:
     url: "http://elasticsearch:9200"
```

Comment:
maybe we should have different config files for running the server in Docker vs on host?  since this setting won't work when running on host.

Reply:
I'm locally trying to see if I can clean up the docker-compose setup, it would be nice to not have 2 custom Dockerfiles and wait logic. 

For an end user, they would just have to provide whatever url does work. I call this out in readme example.